src/Dialog.php

```diff
@@ -222,6 +221,12 @@ final public function isStart(): bool
         return $this->next === 0;
     }
 
+    /** Check if Dialog on the last step */
+    final public function isLastStep(): bool
```

Comment:
BTW: do you have any meaningful use-cases for this method? I usually try to minimize creation of public methods to avoid BC breaks in the future.

Reply:
I use it just for fix, because isEnd is not useful there. We may just use the check $this->next == count($this->steps) - 1; at that place instead of function, but it seems not good and clear. Probably it's better to make this function private and that's all.

Reply:
let's start from private. If someone will need it public - they can create a PR.